src/components/EditorWidgets/List/ListControl.js

```diff
@@ -258,6 +264,7 @@ export default class ListControl extends Component {
     return (
       <div id={forID} className={c(classNameWrapper, 'nc-listControl')}>
         <TopBar
+          allowAdd={field.get('allow_add')}
```

Comment:
Didn't realize we weren't setting this to `true` by default - we'll need to do so, otherwise this is a breaking change.

Reply:
Ok, so should it be more like allowAdd={field.get('allow_add' = true)}?

Reply:
Close: `field.get('allow_add', true)`